### ISSUE TITLE
[Bug][A11Y][Button] Keyboard focus is not visible on button component

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#473]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

Samsun S22

### OS Version

Android 16

### App version

0.6.0 (485)

### Describe the issue

Keyboard focus is not visible on button component

### Expected Behavior

Native focus should be displayed by default

**Comment by AhmedAmineZr** [comment]
this issue will be resolved into : 
- [ ]#578

**Comment by jerome-regnier** [comment]
Option bien présente dans l'App de test et fonctionnelle
👌 OK Design

**Comment by pya35** [comment]
@AhmedAmineZr  
Keyboard focus has been fixed but only partially.
In fact, the focus is not correct after activating the Full Keyboard Access, which is the keyboard feature used for accessibility matters.

without FKA
<img width="156" height="68" alt="" src="https://github.com/user-attachments/assets/419f55d3-5ed2-4101-b7bc-9893cbbb2c02" />

with FKA
<img width="155" height="63" alt="" src="https://github.com/user-attachments/assets/bf8dee88-ad5b-41cd-b80d-9cc7d828ec15" />
